src/sklearn_evaluation/plot/roc.py

```diff
@@ -193,16 +193,13 @@ class ROC(Plot):
         positive rate of predictions with score >= `thresholds[i]`. If
         None, it will be calculated based on y_true and y_score.
 
+    roc_rates_n_classes : list of dictionaries with 'fpr' and 'tpr' keys
```

Comment:
I'm a bit confused as to why we need this extra parameter. In the [old implementation](https://github.com/ploomber/sklearn-evaluation/blob/6b5f41128fa563c21e5ca7ba0201ab2a0bcea6bb/src/sklearn_evaluation/plot/roc.py#L10), the user only needed to provide y_true and y_score, then we'd figure out the rest (for binary and multi-class). Why do we need this extra parameter?